build/webpack/webpack.config.base.js

```diff
@@ -20,6 +20,53 @@ class AccessDependenciesPlugin {
   }
 }
 
+const defines = {
+  BUILDFLAG: ' '
+}
+
+const buildFlagArg = process.argv.find(arg => arg.startsWith('--buildflags='));
+
+if (buildFlagArg) {
+  const buildFlagPath = '--buildflags='.length
```

Comment:
Shouldn't it be `= buildFlagArg.substr('--buildflags='.length)`?
`'--buildflags='` could be moved to a `const` btw.